Client/Pages/Index.razor

```diff
@@ -66,7 +66,10 @@
                             </span>
                             <div class="desc">
                                 <h3>Easy To Use</h3>
-                                <p>Write your code from anywhere using our integrated editor and easily share you component</p>
+                                <p>
+                                    <a href="https://blazorrepl.com/repl">Write your code</a>
```

Comment:
It's better to use the relative path `/repl` so when we click it on localhost or stage, the URL will still lead to the correct REPL page.

Reply:
fixed